Kernel/Arch/riscv64/Timer.cpp

```diff
@@ -49,6 +49,11 @@ void Timer::handle_interrupt()
     set_compare(current_ticks() + m_interrupt_interval);
 }
 
+void Timer::disable()
+{
+    RISCV64::CSR::clear_bits(RISCV64::CSR::Address::SIE, 1 << (to_underlying(CSR::SCAUSE::SupervisorTimerInterrupt) & ~CSR::SCAUSE_INTERRUPT_MASK));
```

Comment:
@Hendiadyoin1 if this PR is merged before #25800, you should maybe also change this to use your new CSR definitions.